bindata/v4.1.0/kube-controller-manager/pod.yaml

```diff
@@ -30,16 +30,16 @@ spec:
             --authentication-kubeconfig=/etc/kubernetes/static-pod-resources/configmaps/controller-manager-kubeconfig/kubeconfig \
             --authorization-kubeconfig=/etc/kubernetes/static-pod-resources/configmaps/controller-manager-kubeconfig/kubeconfig \
             --client-ca-file=/etc/kubernetes/static-pod-certs/configmaps/client-ca/ca-bundle.crt \
-            --requestheader-client-ca-file=/etc/kubernetes/static-pod-certs/configmaps/aggregator-client-ca/ca-bundle.crt
+            --requestheader-client-ca-file=/etc/kubernetes/static-pod-certs/configmaps/aggregator-client-ca/ca-bundle.crt \
+            --pv-recycler-pod-template-filepath-hostpath=/etc/kubernetes/static-pod-resources/configmaps/kube-controller-manager-recycler-config/recycler-pod.yaml \
```

Comment:
@soltysh your call. I don't feel strongly about a move here versus the extended args.  But half and half seems not-awsome.

Reply:
moved to extended args

Reply:
:+1: for extended args